src/modules/listCard.js

```diff
@@ -0,0 +1,16 @@
+import listItems from './data.js';
+
+const toDoListContainer = document.getElementById('to-do-list-container');
+
+const cardDisplay = () => {
+  listItems.forEach((item) => {
+    const toDoListItems = `
+        <ul>
+          <li class="item-list">${item.description}</li>         
+        </ul>
+          `;
+    toDoListContainer.insertAdjacentHTML('afterbegin', toDoListItems);
+  });
+};
```

Comment:
- [ ] Kindly make sure that the todo list is displayed in the order of their indexes, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.) requirement.

Reply:
Noted, I would ensure this is worked on. Thanks for the review